deploy-accounting.sh

```
@@ -0,0 +1,207 @@
+echo -e "${GREEN}✓ Contenedor activo: ${RUNNING_CONTAINER}${NC}"
+echo ""
+
+# ---- 2. Copiar los nuevos archivos PHP al contenedor ----
+echo -e "${YELLOW}[2/5] Copiando archivos del módulo al contenedor...${NC}"
+
+copy_file() {
+    local SRC="$1"
+    local DEST="$2"
+    if [ -f "$SRC" ]; then
+        docker cp "$SRC" "$RUNNING_CONTAINER:$DEST"
+        echo -e "  ${GREEN}✓ Copiado:${NC} $(basename $SRC)"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)